lightstep-tracer-jre/benchmark.gradle

```diff
@@ -0,0 +1,45 @@
+def current_version = new File(project.projectDir, '../common/VERSION').text
```

Comment:
Should probably add `.trim()` to the end of this so when text editors add a newline, the gradle file doesn't break (this change was recently made to all the other gradle files).

```
def current_version = new File(project.projectDir, '../common/VERSION').text.trim()
```